build/sections/header.html

```diff
@@ -1,4 +1,23 @@
-<div>
-    <h1 class="text-primary-green">Header</h1>
-    <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Quisquam, quibusdam.</p>
+<div class="container py-5 flex justify-between items-center">
+    <a href="">
+        <img width="100" src="Images/Ojemba logo.png" alt="">
+    </a>
+    <button class="lg:hidden">
+        <?xml version="1.0" encoding="utf-8"?><!-- Uploaded to: SVG Repo, www.svgrepo.com, Generator: SVG Repo Mixer Tools -->
+        <svg width="40" height="40" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
+            <path d="M4 7L7 7M20 7L11 7" stroke="#fff" stroke-width="1.5" stroke-linecap="round"/>
+            <path d="M20 17H17M4 17L13 17" stroke="#fff" stroke-width="1.5" stroke-linecap="round"/>
+            <path d="M4 12H7L20 12" stroke="#fff" stroke-width="1.5" stroke-linecap="round"/>
+        </svg>
+    </button>
+    <ul class="hidden lg:flex gap-x-12 text-white font-roboto">
+        <li><a href="">Services</a></li>
+        <li><a href="">Über uns</a></li>
+        <li><a href="">Darum wir</a></li>
+        <li><a href="">Team</a></li>
+        <li><a href="">Über Ruanda</a></li>
+        <li><a href="">FAQ</a></li>
+        <li><a href="">Kontakt</a></li>
+        <li><a href="">DE</a></li>
```

Comment:
Kindly check this section. You are missing `EN` 
This is the Figma design.
![Screenshot (19)](https://github.com/UmutesiMMA/ojembaProject/assets/99269832/66e1c0ec-56c8-48e0-8cb1-543112b710b1)

And your Design looks like this: 

![Screenshot (20)](https://github.com/UmutesiMMA/ojembaProject/assets/99269832/ed7638f7-b242-437d-acdf-fbdc82150cf7)

On the medium screen the nav links get spread across the screen and reach near the logo. I would suggest that you reduce the space between the links slightly so that they extent to the right. 
![Screenshot (18)](https://github.com/UmutesiMMA/ojembaProject/assets/99269832/f5bc2e56-2981-44e0-a886-798bbce567ff)

Reply:
Thanks for the feedback let work on it👍